### ISSUE TITLE
Update ansi-regex

### ISSUE THREAD

**Issue by massongit** [azu/nlp-pattern-match#12]
`ansi-regex` 2.1.1 ~ 5.0.0 is vulnerable to Inefficient Regular Expression Complexity: CVE-2021-3807
Therefore, you need to update it to 5.0.1.

**Comment by massongit** [comment]
```
$ yarn audit --groups dependencies
yarn audit v1.22.11
0 vulnerabilities found - Packages audited: 32
✨  Done in 1.01s.
```

There is no vulnerable package.
Therefore, I close it.

**Comment by azu** [comment]
Publish new versions for each package.
Also I will publish again these as new fixed version #16

**Comment by azu** [comment]
all packages are updated to 1.4.0
